docs/user/openstack/README.md

```diff
@@ -48,7 +48,7 @@ enough to store the ignition config files, so they are served by swift instead.
 `openstack image create --container-format=bare --disk-format=qcow2 --file rhcos-${RHCOSVERSION}-openstack.qcow2 rhcos-${RHCOSVERSION}`
 
 **NOTE:** Depending on your OpenStack environment you can upload the RHCOS image
-as `raw` or `qcow2`. See [Disk and container formats for images](https://docs.openstack.org/image-guide/image-formats.html) for more information.
+as `raw` or `qcow2`. See [Disk and container formats for images](https://docs.openstack.org/image-guide/image-formats.html) for more information. And the imaeg by default must use `rhcos` as prefix.
```

Comment:
Spelling: image

From https://github.com/openshift/installer/blob/25d1ae28d11c71b96fb8b8de1095911923d9d8ee/pkg/asset/rhcos/image.go#L43 I understand we hardcode the image name for openstack to `rhcos` with the possiblity to override via the `OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE` environment variable.

Now, I'm not sure I'm looking at the right file :)

Reply:
ok, I didn't notice OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE ,I will dig this and update the doc as well